### PR TITLE
Fix errors in curated stat var descriptions

### PR DESCRIPTION
**Pull request by shifucun**
sv diff report: https://storage.mtls.cloud.google.com/datcom-embedding-diffs/boxu_base_uae_mem_2024_06_20_22_01_06.html

### CODE REVIEW

File: tools/nl/embeddings/data/curated_input/base/_preindex.csv [hunk 1]
```diff
@@ -596,9 +596,8 @@ Count of Earthquake Event: 7 - 8 Magnitude,Count_EarthquakeEvent_M7To8
 Count of Earthquake Event: 8 - 9 Magnitude,Count_EarthquakeEvent_M8To9
 Count of Earthquake Event: 9 Magnitude or More,Count_EarthquakeEvent_M9Onwards
 Count of Establishment,Count_Establishment
-Count of households,Count_Household
```
Comment (pradh) [inline tools/nl/embeddings/data/curated_input/base/_preindex.csv:599]: why is this deleted?

Comment (pradh) [reply tools/nl/embeddings/data/curated_input/base/_preindex.csv:599]: oh, its now number of households, got it!

**Comment by pradh** [comment]
In Q34, why do we lose `Count_Household_HouseholderAge65OrMoreYears_MarriedCoupleFamilyHousehold` ?

Other than that the SV Differ diffs look good.

**Comment by shifucun** [comment]
> In Q34, why do we lose `Count_Household_HouseholderAge65OrMoreYears_MarriedCoupleFamilyHousehold` ?
> 
> Other than that the SV Differ diffs look good.

the change is from `married-couple` to `married couple`. The old score is 0.701 which is also very borderline..

File: tools/nl/embeddings/data/curated_input/base/sheets_svs.csv [hunk 2]
```diff
@@ -2511,7 +2497,7 @@ Count_Person_NonWorker_Urban,Number of urban people who do not work
 Count_Person_NotAUSCitizen,Number of non US citizens
 Count_Person_NotEnrolledInSchool,Number of people not enrolled in school
 Count_Person_NotInLaborForce,Number of people Aged 16 and Over Not in the Labor Force
-Count_Person_NowMarried,Number of married females
+Count_Person_NowMarried,Number of married (including separated) people
```
Comment (pradh) [inline tools/nl/embeddings/data/curated_input/base/sheets_svs.csv:2500]: nit: "Number of married people, including separated"

Not sure how having something in `()` affects

**Review by pradh — APPROVED** [review]
Very nice bunch of changes!

File: tools/nl/embeddings/data/curated_input/base/sheets_svs.csv (hunk 2 above)
Comment (shifucun) [reply tools/nl/embeddings/data/curated_input/base/sheets_svs.csv:2500]: okay, i will update in following PR, so not re-generate the embeddings again.

Comment (pradh) [reply tools/nl/embeddings/data/curated_input/base/sheets_svs.csv:2500]: makes sense!

**Comment by shifucun** [comment]
thanks for review!

File: server/integration_tests/test_data/e2e_india_demo/whichstatesinindiahavethehighestpovertylevelspercapita/chart_config.json [hunk 3]
```diff
@@ -297,118 +297,82 @@
...
-              },
-              {
-                "tiles": [
-                  {
-                    "statVarKey": [
-                      "sdg/NY_GDP_PCAP"
-                    ],
-                    "title": "Annual Growth Rate of Real GDP Per Capita in Administrative Area 1 Places of India (${date})",
-                    "type": "MAP"
+                    "title": "Employed Population 15 to 24 Years Old Below International Poverty Line, by Sex (%) in India",
```
Comment (pradh) [inline server/integration_tests/test_data/e2e_india_demo/whichstatesinindiahavethehighestpovertylevelspercapita/chart_config.json:305]: It seems this is better than LHS for poverty query, so find losing sdg/NY_GDP_PCAP

File: server/integration_tests/test_data/international/query_3/chart_config.json [hunk 4]
```diff
@@ -93,35 +93,6 @@
             ],
             "title": "Nominal GDP Per Capita"
           },
-          {
-            "columns": [
-              {
-                "tiles": [
-                  {
-                    "statVarKey": [
-                      "sdg/NY_GDP_PCAP"
```
Comment (pradh) [inline server/integration_tests/test_data/international/query_3/chart_config.json:102]: Why do we lose this for GDP query?

Comment (shifucun) [reply server/integration_tests/test_data/international/query_3/chart_config.json:102]: with growth rate and per capita,  this seems not that close to the query anymore
